pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -75,7 +131,14 @@ metadata:
   namespace: {{.eksaSystemNamespace}}
 spec:
   template:
+    {{- if and .controlPlanetemplateOverride (ne .format "")}}
+    spec:
+      templateOverride: |
+{{.controlPlanetemplateOverride | indent 8}}
+    {{- end }}
+    {{- if (eq .controlPlanetemplateOverride "") }}
     spec: {}
+    {{- end }}
```

Comment:
I think we always want to pass templateOverride and not rely on the default CAPT template